public/js/chat.js

```diff
@@ -235,13 +246,9 @@ var VideoChat = {
 					// Arbitrary data handling
 					console.log("Received arbitrary data: ", receivedData);
 					document.getElementById("arbitrary-data").append(receivedData);
+					window.top.postMessage(receivedData, "*");
```

Comment:
Should we add a type field for the data that we're posting to the parent as well? Just in case the application consuming the video chat is also listening to other messages from other iframes.